test/hotspot/jtreg/compiler/loopopts/superword/RedTest_int.java

```diff
@@ -157,6 +141,12 @@ public static void reductionInit2(
     }
 
 
+    @Test
+    @IR(applyIfOr = {"SuperWordReductions", "false", "LoopMaxUnroll", "< 8"},
+        failOn = {IRNode.ADD_REDUCTION_VI})
+    @IR(applyIfCPUFeature = {"sse4.1", "true"},
+        applyIfAnd = {"SuperWordReductions", "true", "LoopMaxUnroll", ">= 8"},
+        counts = {IRNode.ADD_REDUCTION_VI, ">= 1"})
```

Comment:
We could also remove the store to `d` here. I think it should still vectorize.
Or maybe have an additional test where we do no store.